staging/src/k8s.io/kubectl/pkg/cmd/diff/diff.go

```diff
@@ -131,6 +137,10 @@ func NewDiffOptions(ioStreams genericclioptions.IOStreams) *DiffOptions {
 			Exec:      exec.New(),
 			IOStreams: ioStreams,
 		},
+		pruner: &pruner{
```

Comment:
Maybe make a constructor for the pruner? That would avoid it from being mis-used